### ISSUE TITLE
List of restrictions in path details

### ISSUE THREAD

**Issue by thomfre** [graphhopper/graphhopper#1260]
It would be nice if we could request a list of restrictions (seasonal closure, controlled border crossings etc) in the path details, to warn the user about possible road closures or expected delays.

**Comment by boldtrn** [comment]
I agree this would be interesting. Border crossings could be done by checking for [`barrier=border_control`](https://wiki.openstreetmap.org/wiki/Tag:barrier=border_control). In Europe most border_control tags some border control tags are still used, even though they are not active anymore, which could lead to unexpected results. Seasonal closure with conditional access values.
